### PR TITLE
Jobify annotation

### PR DESCRIPTION
**Pull request by adesca**
Adds annotation that executes a function in the background

### CODE REVIEW

File: invisible_flow/app_factory.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from flask import Flask
```
Comment (tw-jeff-burroughs) [inline invisible_flow/app_factory.py]: I'm curious about the need for creating this AppFactory, is there something in particular that this change makes better for us?

File: manage.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# # SQLALCHEMY_DATABASE_URI = 'postgres://localhost/main'
```
Comment (tw-jeff-burroughs) [inline manage.py]: I typically try to remove any commented out code if it's not going to be used later.

File: tests/jobs/test_jobify.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
+from unittest.mock import patch, Mock
+
+from invisible_flow.jobs.jobify import jobify
+
+
+def function_to_wrap(a):
+    return a + 10
+
+
+# @patch('invisible_flow.jobs.jobify.db')
```
Comment (tw-jeff-burroughs) [inline tests/jobs/test_jobify.py]: I would remove this comment too, if it's not saved for a particular reason

File: invisible_flow/app_factory.py (hunk 1 above)
Comment (adesca) [reply invisible_flow/app_factory.py]: By pulling out the creation of the app object it becomes easier to test things that use it without loading the entire context. In this particular commit, it allowed me to test the entities and their sqlalchemy validations

File: manage.py (hunk 2 above)
Comment (adesca) [reply manage.py]: I'll remove it

File: tests/jobs/test_jobify.py (hunk 3 above)
Comment (adesca) [reply tests/jobs/test_jobify.py]: Removed

**Pull request merged**